src/content/overview/observability/observability-platform-api/_index.md

```diff
@@ -0,0 +1,105 @@
+description: Documentation on the observability-platform API architecture deployed and maintained by Giant Swarm.
+weight: 80
+menu:
+  principal:
+    identifier: overview-observability-platform-api
+    parent: overview-observability
+user_questions:
+  - What is the observability-platform API architecture?
+  - What is the use of observability-platform API?
+  - How may I use the observability-platform API?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2025-02-10
+---
+
+The following page describes the concept of the Observability Platform API
+
+## Table of contents
+
+* [Context](#context)
+* [Architecture](#architecture)
+* [Global overview](#global-overview)
+  * [How it works](#how-it-works)
+    * [Read path](#read-path)
+    * [Write path](#write-path)
+* [Adding datasources](#adding-datasources)
+* [Limitations](#limitations)
+
+## Context
+
+The observability platform allows customers to ingest their observability data (metrics and logs) from inside the clusters into our platform by default.
+
+The `observability platform API` is an additional component that is a part of the platform, and it allows customers to ingest their logs (for now) from outside their clusters into our observability platform. Moreover, it also allows them to access our own observability data through their own Grafana running on their premises.
+
+Its main objectives are to be able to:
+
+* provide a secure access to our observability platform and reduce the attack surface.
+* allow customers to access our observability data and to send their own.
+* sanitize (sampling, relabelling, ...) observability data before it reaches our backends.
+* support OTLP (OpenTelemetry).
+
+## Architecture
+
+### Global overview
+
+Here is a graph showing the general architecture for the API :
+
+![api architecture](./observability-platform-api-graph.png)
+
+### How it works
+
+Let's explore the 2 main workflows from the customer's perspective.
+
+#### Read path
+
+Let's consider the following situation : a customer wants to access Giant Swarm observability data from his own Grafana running on his premises.
+
+To do so, the customer will first log into the Grafana instance using the OIDC provider hosted on his premises. Once authenticated, he will perform a query from the "Explore" panel or simply use dashboards which in any case will trigger a request sent to the Observability Platform API url.
+
+There, depending on the path the requests is trying to access (i.e either trying to access logs or metrics) it will go through one of 2 ingresses sharing the same host : one in the loki namespace and the other in the mimir one. Note that we decided to have 2 ingresses sharing the same host instead of having 1 ingress handling all paths because usually, ingresses only forward requests in the same namespace they are deployed in. There are ways to go around this, but those are a bit tricky and definitely not straight forward.
```

Comment:
I think this is a very technical detail that adds only limited value for the customer, but adds "more text" to go through, if you know what i mean? Customers usually read through docs more interested in "getting something done" instead of needing to know every detail. Especially that something is tricky is not interesting for customers. That's why they pay us money to not worry about the tricky bits :) So let's leave that out, wdyt?